app/keepers/keepers.go

```diff
@@ -338,6 +341,16 @@ func (appKeepers *AppKeepers) InitNormalKeepers(
 	)
 	appKeepers.TokenFactoryKeeper = &tokenFactoryKeeper
 
+	validatorPreferenceKeeper := validatorpreference.NewKeeper(
+		appKeepers.keys[validatorpreferencetypes.StoreKey],
+		appKeepers.GetSubspace(validatorpreferencetypes.ModuleName),
+		appKeepers.StakingKeeper,
+		appKeepers.BankKeeper,
+		appKeepers.DistrKeeper,
```

Comment:
Can we remove passing this in?

Reply:
Yeah, I'm confused as to why we need `DistrKeeper` or `BankKeeper`? I can't find references to where they're used.

Reply:
They were added to be used by upcoming keeper methods, but i can remove them now and add them later :)